src/mf/mf-hid.cpp

```diff
@@ -167,11 +167,9 @@ namespace librealsense
                     CHECK_HR(report->GetSensorValue(SENSOR_DATA_TYPE_ANGULAR_VELOCITY_Z_DEGREES_PER_SECOND, &var));
                     rawZ = var.dblVal;
 
-                    static constexpr double gyro_transform_factor = 10.0;
-
-                    rawX *= gyro_transform_factor;
-                    rawY *= gyro_transform_factor;
-                    rawZ *= gyro_transform_factor;
+                    rawX *= _gyro_scale_factor;
```

Comment:
Add a TODO - get scale factor from HID descriptor, then setting it from the feature will not be needed.

Reply:
Added

Reply:
As discussed, we can't access the value from the descriptor. Can remove the TODO.

Reply:
Removed